charts/spire/templates/server-statefulset.yaml

```diff
@@ -88,6 +88,31 @@ spec:
             - name: spire-workload-registrar-config
               mountPath: /run/spire/k8s-workload-registrar/config
               readOnly: true
+        - name: spire-oidc
+          image: "{{ .Values.oidc.image.repository }}:{{ .Values.oidc.image.tag | default .Chart.AppVersion }}"
+          args:
+            - -config
+            - /run/spire/oidc/config/oidc-discovery-provider.conf
+          ports:
+            - containerPort: 443
+              name: spire-oidc-port
+          volumeMounts:
+            - name: spire-server-socket
+              mountPath: {{ include "spire.sockets" . }}
```

Comment:
Depending on the booting order of this Pod (3 containers) this socket might not exist yet. 

In the spire-oidc container there is a warning about this.